packages/cdktf/lib/terraform-local.ts

```diff
@@ -40,6 +40,22 @@ export class TerraformLocal
     return Token.asList(this.interpolation());
   }
 
+  public get asStringMap(): { [key: string]: string } {
```

Comment:
We missed the feature, so one of our TS examples was just impossible to achieve in Python.

Reply:
Good catch! As the fix won't show up in the generated changelog, can we add a small manual section to the changelog file that notes that this has been added (I don't think we need to call it breaking, as the 0.13->0.14 change will already "break" older pre-built providers.

Reply:
Yeah makes sense 👍